app/controllers/orders_controller.rb

```diff
@@ -11,11 +11,30 @@ def create
       redirect_to cart_path
     else
       order = OrderCreator.new.create(current_user, @cart.contents)
+
+      @amount = @cart.total_price * 100
+
+      customer = Stripe::Customer.create(
+        :email => params[:stripeEmail],
+        :source  => params[:stripeToken]
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Operator `=>` should be surrounded by a single space.